src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -235,6 +236,14 @@ public function drupalLoginLink(
     ): Result {
         $this->io()->section("create login link.");
         $uid = $this->getDrupalSiteAdminUid(siteName: $siteDir);
+        if ($environmentType === 'ddev') {
+            return $this->taskExec('drush')
+                ->arg("@$siteDir.$environmentType")
+                ->arg('user:login')
+                ->option("--uid=$uid")
+                ->dir("$this->drupalRoot/sites/$siteDir")
```

Comment:
@apotek Is this line needed? Running drush using the alias in `ln 241` should take care of hitting the correct site anyway? (ie. `drush @paypal.ddev user:login --uid=1`)

Reply:
@mmatsoo Good catch and good question! I wasn't sure why we were specifying both `--dir` and a drush alias, since the drush alias should be sufficient. 

But those were both there in the original code, and I decided to focus on removing the `ddev` wrapper rather than change the implementation, since I was aware that this functionality is fundamental to a lot of developer workflows and I was afraid to make the change. It's already going to cause some grief when developers have to remember to run this in ddev instead of on their host OS. That will be a pain-point for a while, but will keep us moving in the right direction of no longer having to worry about the host OS php version or composer version or drush version at all.